pkg/querier/worker/worker_test.go

```diff
@@ -17,46 +18,52 @@ import (
 
 func TestResetConcurrency(t *testing.T) {
 	tests := []struct {
-		name                string
-		parallelism         int
-		maxConcurrent       int
-		numTargets          int
-		expectedConcurrency int
+		name                                  string
+		parallelism                           int
+		maxConcurrent                         int
+		numTargets                            int
+		expectedConcurrency                   int
+		exepctedConcurrencyAfterTargetRemoval int
```

Comment:
Typo fix:
```suggestion
		expectedConcurrencyAfterTargetRemoval int
```

Reply:
oh my god, thank you so much for catching it!